assets/js/types/type-defs/payment-method-interface.ts

```diff
@@ -166,6 +166,7 @@ export type PaymentMethodInterface = {
 	// Various payment status helpers.
 	paymentStatus: {
 		isPristine: boolean;
```

Comment:
Should this be removed (pending decision on idle vs pristine).

Reply:
I've deprecated it in the use-payment-interface, but can't remove it just yet.